Xamarin.Forms.Platform.WPF/Renderers/EditorRenderer.cs

```diff
@@ -2,6 +2,7 @@
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Media;
+using WpfScrollBarVisibility = System.Windows.Controls.ScrollBarVisibility;
```

Comment:
Needed to alias one of ScrollBarVisibility enums, decided to do that to the platform enum instead of forms